hw_4_5.js

```diff
@@ -0,0 +1,277 @@
+"use strict"
+
+//Task 1
+
+// let sum = 0;
+// function getSum(x) {
+//     for (let i = 0; i <= x; i++) {
+//         sum += i;
+//     }
+
+//     console.log(sum);
+// }
+
+// getSum(100);
+
+//Task 2
+
+const getCredit = function(sumOfCredit) {
+    let needToReturn = sumOfCredit * 1.17;
+    let overPaid = needToReturn - sumOfCredit;
+    console.log(overPaid);
+}
+
+getCredit(5000);
+
+//Task 3
+
+// function trimString(string, from, before) {
+//     console.log(string.slice(from, before));
+// }
+
+// trimString('JS лучший язык', 3, 9);
+
+//Task 4
+
+let sum = 0;
+function getSumNumbers(chislo) {
+    let arr = chislo.toString().split('');
+    for (let i = 0; i < arr.length; i++) {
+        let arrChisla = Number(arr[i]);
+        sum += arrChisla;
+    }
+    console.log(sum);
+}
+
+getSumNumbers(2021); 
+
+//Task 5
+
+// let sum = 0;
+// function getSum(a, b) {
+//     while (a <= b) {
+//         sum += a;
+//         a++;
+//     }
+//     console.log(sum);
+// }
+
+// getSum(1, 5);
```

Comment:
а если я укажу 
```suggestion
// getSum(5, 1);
```
выведет 0, что неверное